src/utils/resources/vm/utils/network/constants.ts

```diff
@@ -41,7 +41,7 @@ const labels2types: LabelMap = Object.fromEntries(
   ),
 );
 
-export const interfacesTypes = new Proxy<TypeMap>(types2labels, typeHandler);
+export const interfaceTypes = new Proxy<TypeMap>(types2labels, typeHandler);
```

Comment:
that's a good change but has effects all across the code - for this single reason I would extract this to a separate PR.